e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -272,4 +277,44 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         // * Verify the content and message in next screen
         verifyInvitationSuccess(email, 'An invitation email has been sent.');
     });
+
+    it('MM-18050 Verify when different feature settings are disabled', () => {
+        // # Disable Guest Account Feature
+        changeGuestFeatureSettings(false, true);
+
+        // # Login again after changing feature flag and reload current page
+        cy.apiLogin('sysadmin');
+        cy.visit(`/${testTeam.name}`);
+
+        // # Open Invite People
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+        cy.get('#invitePeople').should('be.visible').click();
+
+        // * Verify if Invite Members modal is displayed when guest account feature is disabled
+        cy.getByTestId('invitationModal').within(($el) => {
```

Comment:
This can become a one liner:

`cy.getByTestId('invitationModal').find('h1').should('have.text', 'Invite Members')`